conda_lock/conda_lock.py

```diff
@@ -745,13 +813,32 @@ def create_lockfile_from_spec(
         for dep in deps:
             locked[(dep.manager, dep.name, dep.platform)] = dep
 
+    git_metadata = GitMeta.create() if add_git_metadata else None
+    time_metadata = TimeMeta.create() if add_time_metadata else None
+    inputs_metadata: Optional[Dict[str, InputMeta]] = (
+        {str(src_file): InputMeta.create(src_file=src_file) for src_file in src_files}
```

Comment:
This should store the name as a relative path (maybe relative to the git root?) not absolute one